conda/recipes/rapids-dask-dependency/meta.yaml

```diff
@@ -28,10 +28,9 @@ requirements:
     - setuptools
     - conda-verify
   run:
-    - dask ==2024.12.1
-    - dask-core ==2024.12.1
-    - distributed ==2024.12.1
-    - dask-expr ==1.1.21
+    - dask >=2025.1.0
+    - dask-core >=2025.1.0
+    - distributed >=2025.1.0
```

Comment:
Note that we drop the `dask-expr` dependency, because that module is included within `dask.dataframe` for `2025.1.0+`.

Reply:
dask-cudf should support `dask >=2024.12.1`. However, it may be messy to support pre- and post- dask-expr migration at the same time.

Reply:
Yes, this is aligned with our previous decisions, we support _one_ Dask version at a time because it's too consuming to support more.